autometa/config/user.py

```diff
@@ -295,17 +302,39 @@ def run_binning(self, mgargs):
             index=True,
             header=True)
 
-def main(args):
-    logger.info(args.user)
-
-if __name__ == '__main__':
+def main():
     import argparse
+    import multiprocessing as mp
     import logging as logger
+    parser = argparse.ArgumentParser(description="""
+    Configures the Autometa user environment/databases.
+    Running without args will download and format Autometa database dependencies.
+    """)
+    parser.add_argument('--config',
+        help=f'</path/to/user.config> (default is {config.DEFAULT_FPATH}).',
+        default=config.DEFAULT_FPATH)
+    parser.add_argument('--dryrun',
+        help='Log configuration without performing updates.',
+        action='store_true',
+        default=False)
+    parser.add_argument('--debug',
+        help='Stream debugging information to terminal.',
+        action='store_true',
+        default=False)
+    parser.add_argument('--cpus',
+        help=f'</path/to/user.config> (default is {mp.cpu_count()}).',
```

Comment:
Wrong help message here - this is not a path but an int. Should be something like "<number of threads to use>"